src/components/custom/other-platforms.js

```diff
@@ -556,18 +556,20 @@ export const NavResources = ({ onClick }) => (
             onClick={onClick}
             to="/payment-methods/"
         />
-        <CardLink
-            icon={() => <img src={Signals} alt="" width="24" height="24" />}
-            title={localize('DMT5 Signals')}
-            onClick={onClick}
-            to="/dmt5-trading-signals/#signal-subscriber/"
-        />
-        <CardLink
-            icon={() => <img src={Blog} alt="" width="24" height="24" />}
-            title={localize('Academy')}
-            onClick={onClick}
-            to="/academy/"
-        />
+        <ROW>
+            <CardLink
+                icon={() => <img src={Signals} alt="" width="24" height="24" />}
+                title={localize('DMT5 Signals')}
+                onClick={onClick}
+                to="/dmt5-trading-signals/#signal-subscriber/"
+            />
+            <CardLink
+                icon={() => <img src={Blog} alt="" width="24" height="24" />}
+                title={localize('Academy')}
+                onClick={onClick}
+                to="/academy/"
+            />
+        </ROW>
```

Comment:
Are these only supposed to be available for ROW?

Reply:
Yes, QA asked for this.
https://redmine.deriv.cloud/issues/58762#Remove_DMT5_signals_and_Academy_for_EU/UK_from_header

Reply:
But we will need to also remove from footer and mobile side menu 👀 AFAIK, Academy isn't supposed to be hidden from EU/UK. That's why we haven't content based on geolocation.

Reply:
According to Figma Design, we have it in footer but we don't have it in nav. Can we discuss it with designers?

Reply:
Designers and QA asked to keep Signals and Academy in EU UK nav. Now it's back.